viewer/config.js

```diff
@@ -433,6 +433,11 @@ exports.hostName = function () {
   return internals.hostName;
 };
 
+exports.arkimeWebURL = () => {
+  const webUrl = exports.get('arkimeWebURL', `${exports.hostName()}${exports.basePath()}`);
+  return exports.isHTTPS() ? `https://${webUrl}` : `http://${webUrl}`;
```

Comment:
so if arkimeWebURL is used it should either not add http/https or only if doesn't start with http since user might have a reverse proxy in front.